src/controllers/PhotoController.ts

```diff
@@ -16,6 +16,28 @@ class PhotoController implements IController {
     const photo = await this.photoService.create(photoPayload);
     res.status(201).json({ data: photo });
   };
+
+  public updateById: IRequestHandler = async (req, res) => {
+    const photoId = Number(req.params.id);
+    const photoPayload = req.body;
+    const photo = await this.photoService.updateById(photoId, photoPayload);
+    if(photo){
+      res.status(200).json({ data: photo });
+    } else {
+      res.status(404).json({ error: photo });
```

Comment:
We want to handle this from an error middleware instead, otherwise, we'll end up with a lot of repeated code across controllers only for handling not found entities. Please read the other comments and see how we are handling this in my own master branch 😃